### DISCUSSION TITLE
Game One

### DISCUSSION THREAD

**Discussion by baliebox4916** [iptv-org/community#829] [Q&A]
### Channel ID

GameOne.fr

### Notes

Some Other Channel Requests

This Time Regarding Anime

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19817
